app/controllers/blogs_controller.rb

```diff
@@ -0,0 +1,59 @@
+class BlogsController < ApplicationController
+
+  before_action :logged_in_user, only: [:new, :edit, :index, :show, :create,]
+
+
+  def index
+
+  end
+
+
+  def show
+
+  end
+
+
+  def edit
+
+  end
+
+  def new
+  	@blog = Blog.new
+  end
+
+  def create	
+  	@blog = Blog.new(params.require(:blog).permit(:title,:caption,:description))
```

Comment:
I made a function for this.
But I don't know how to simplify in order to remove require and permit .